src/mixins/crownConfig.js

```diff
@@ -69,6 +79,9 @@ export default {
     shapeView() {
       return this.shape.findView(this.paper);
     },
+    selectShape() {
+      return this.shapeView.$el.find('[joint-selector=body]');
+    },
```

Comment:
I think the name of this could be improved. In my opinion, `selectShape` sounds like a verb (_select_ the shape), or a shape that is of type "select". What about a name like `shapeBody`?